### DISCUSSION TITLE
Reduce lookup time by limiting dependencies to Maven repositories?

### DISCUSSION THREAD

**Discussion by Maddin-M** [renovatebot/renovate#26378] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.105.0

### If you're self-hosting Renovate, select which platform you are using.

Gitea or Forgejo

### What is your question?

Hello!

I have currently run into the problem, that one of the repositories I use in my project has shut down or has moved. Therefore Renovate runs into a 404 when looking up for updates for those dependencies. The issue with this is, that it then looks for that dependency in every other possible repository, which is very time-consuming with 50+ dependencies being looked up in 10+ repositories.

In the logs section I have attached an example of the logs that I get, even 30+ minutes after the run has started.

- Is there a way to limit certain dependencies to one (or a group of) repositories?
- Or is there a way to generally better handle this?

Thank you!

### Logs (if relevant)

<details><summary>Logs</summary>

```

gitea-renovate  | DEBUG: Content is not found for Maven url (repository=<my-repo>)
gitea-renovate  |        "url": "https://teavm.org/maven/repository/io/github/libktx/ktx-inject/maven-metadata.xml",
gitea-renovate  |        "statusCode": undefined

```

(The dependency `libktx/ktx-inject` is being looked for in `https://teavm.org/maven/repository/` which will never fetch a result)

</details>

**Comment by rarkins** [comment]
You seem to be referring to Maven specifically. Please update your title/description to reflect that
